packages/SwingSet/test/test-vat-env.js

```diff
@@ -18,20 +19,36 @@ function makeThingInstance(_state) {
   };
 }
 
-test('makeKind is in the vat environment', t => {
-  // TODO: configure eslint to know that makeKind is a global
+test('kind makers are in the vat environment', t => {
```

Comment:
This makes sense, but it was easier to put these in the same test file since they are conceptually related and tests that run in a vat are launched from test files that import `tools/prepare-test-env-ava.js` anyway.